src/LessMsi.Cli/Program.cs

```diff
@@ -43,10 +43,12 @@ enum ConsoleReturnCode
 			UnrecognizedCommand=-3
 		}
 
-		/// <summary>
-		/// The main entry point for the application.
-		/// </summary>
-		[STAThread]
+        private const string c_TempFolderSuffix = "_temp";

```

Comment:
Per https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

```suggestion
        private const string TempFolderSuffix = "_temp";

```